app/views/steps/shared/_approx_date_checkbox.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= form.govuk_check_boxes_fieldset attribute, small: true, legend: { tag: 'span', size: 's', hidden: true } do %>
```

Comment:
Just a few changes to fully be compatible with the new version.

We need to add the `multiple: false` to the `govuk_check_boxes_fieldset`
as per documentation, otherwise the hidden field will have an incorrect name:

https://govuk-form-builder.netlify.app/form-elements/checkboxes/#generating-a-single-checkbox

And we also now can `nil` the legend tag so it does not generate an `h1` (nor a `span` like before).

```suggestion
<%= form.govuk_check_boxes_fieldset attribute, multiple: false, small: true, legend: { tag: nil, size: 's', hidden: true } do %>
```